lib/govspeak.rb

```diff
@@ -89,6 +98,15 @@ def extracted_links(website_root: nil)
       Govspeak::LinkExtractor.new(self, website_root: website_root).call
     end
 
+    def extract_contact_content_ids
```

Comment:
As discussed offline - there doesn't seem to be a clear case for reusing this a lot and the arguments to do so are tricky (since they don't match the markdown syntax)

We've kept the UUID regex so that other things don't have to worry about these not being UUIDs - since they can be anything - but as discussed I've moved it it to a constant (which the linter made me freeze - I didn't even realise a regex was mutable in Ruby)